STL_Extension/include/CGAL/STL_Extension/internal/Has_features.h

```diff
@@ -6,21 +6,17 @@
 //
 // $URL$
 // $Id$
-// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+// SPDX-License-Identifier: LGPL-3.0-or-later OR LicenseRef-Commercial
 //
 // Author(s)     : Stéphane Tayeb, Laurent Rineau
 
-#ifndef CGAL_MESH_3_HAS_FEATURES_H
-#define CGAL_MESH_3_HAS_FEATURES_H
-
-#include <CGAL/license/SMDS_3.h>
-
+#ifndef CGAL_HAS_FEATURES_H
+#define CGAL_HAS_FEATURES_H
 
 #include <boost/mpl/has_xxx.hpp>
 #include <CGAL/tags.h>
 
 namespace CGAL {
-namespace Mesh_3 {
```

Comment:
the name is really generic for something living in CGAL

Reply:
it is inside `internal`

Reply:
The class has indeed a bad name but there is currently an inconsistency in all `Has_XXX` and I left it for the one that will do the uniformisation (which is out of the scope of this branch)